renku/domain_model/dataset.py

```diff
@@ -491,9 +492,9 @@ def derive_from(
         self.initial_identifier = dataset.initial_identifier
         self.derived_from = Url(url_id=dataset.id)
         self.same_as = None
-        self.date_created = date_created or local_now()
+        self.date_created = date_created or dataset.date_created
         self.date_modified = local_now()
-        self.date_published = None
+        self.date_published = dataset.date_published
```

Comment:
Did we decide to keep `date_published` on the derivatives of an imported dataset? I don't recall this.

Reply:
I had a discussion with KG on this and we thought it would be consistent. date_created or date_published are kept on each dataset on the chain, so one does not have to go up the chain to know these things, which can improve some queries. The other option would be to remove both properties on all derivations/set to `None`.

It didn't seem like it'd cause an issue so it seemed sensible to me.